client-sq/src/components/Dashboard.js

```diff
@@ -181,8 +198,21 @@ function Dashboard(){
               borderBottom: "none" }
           }}
         >
-          <TableRow style={{height:window.innerHeight*0.35}}>
+          <TableRow style={{height:window.innerHeight*0.3}}>
             <h2 style={{color:"#3d435a"}}>Now Playing</h2>
+              <div style={{display:"inline-flex", width:"90%"}}>
```

Comment:
What are your thoughts on wrapping this div section into a "NowPlaying" component? We can update it when we have the playbackState API set up properly.